app/services/contributors_generator.rb

```diff
@@ -16,14 +16,24 @@ def self.form_array_from_contributor_event(work:)
     new(work: work).form_array_from_contributor_event
   end
 
+  sig do
+    params(work: Work, pub_date: T.nilable(Cocina::Models::Event))
+      .returns(T::Array[Cocina::Models::Event])
+  end
```

Comment:
personally, i find doing this for "line too long" rubocop exceptions to be annoying and to make it harder to identify the actual method, but I'm playing along with the team.

Reply:
I kinda prefer this arrangement of breaking up the `params` and the `returns`, but I don't care too much either way.

Reply:
that doesn't bother me -- it's the "do" and "end" that make it look like a method.